app/assets/stylesheets/users.scss

```diff
@@ -469,7 +469,7 @@
 }
 
 .notification-container {
-  width: 558px;
+  width: 100%;
```

Comment:
In desktop, it's not looking good. Try to take responsive approach by media query or available bootstrap classes